cookbooks/aws-parallelcluster-environment/recipes/config/ebs.rb

```diff
@@ -27,24 +27,12 @@
 
   # Mount each volume with NFS
   shared_dir_array.each do |dir|
-    dirname = format_directory(dir)
-    exported_dirname = format_directory(dir)
-
-    # Created shared mount point
-    directory dirname do
-      mode '1777'
-      owner 'root'
-      group 'root'
-      recursive true
-      action :create
-    end
-
-    # Mount shared volume over NFS
-    mount dirname do
-      device(lazy { "#{node['cluster']['head_node_private_ip']}:#{exported_dirname}" })
+    volume "mount volume #{index}" do
+      action :mount
```

Comment:
Wouldnt we need enable action too?

Reply:
We're calling the mount of the `volume` resource that includes both `mount` and `enable` of the `nfs` resource.